config/v1/types_oauth.go

```diff
@@ -583,3 +583,35 @@ type OAuthList struct {
 
 	Items []OAuth `json:"items"`
 }
+
+// OAuthSpec contains desired cluster auth configuration
+type OAuthSpec struct {
```

Comment:
I think what you wanted to do is to extend the already existing `type OAuthSpec struct` at the top ([link](https://github.com/openshift/api/blob/4ce104f50a7171b24e9bb0d8470e8955ab52f930/config/v1/types_oauth.go#L30)).

Reply:
Fixed